config/routes.rb

```diff
@@ -1,5 +1,9 @@
 Rails.application.routes.draw do
 
+  get 'bookings/index'
```

Comment:
we defined all the routes in the resources below so you should delete all the newly created routes when you generated the controller